pyperf/_process_time.py

```diff
@@ -61,7 +61,7 @@ def bench_process(loops, args, kw, profile_filename=None):
         args = [args[0], "-m", "cProfile", "-o", temp_profile_filename] + args[1:]
 
     for _ in range_it:
-        start_rss = get_max_rss()
+        start_rss = get_max_rss(resource.RUSAGE_CHILDREN)
```

Comment:
`resource.RUSAGE_CHILDREN` doesn't work on Windows: resource module is not available there.

You should add an abstraction, like having a children=True parameter for get_max_rss().